healthcareai/trained_models/trained_supervised_model.py

```diff
@@ -24,20 +26,35 @@ def __init__(self,
                  column_names,
                  grain_column,
                  prediction_column,
-                 y_pred,
-                 y_actual,
+                 test_set_predictions,
+                 test_set_actual,
                  metric_by_name):
         self.model = model
         self.feature_model = feature_model
         self.fit_pipeline = fit_pipeline
         self.column_names = column_names
-        self.model_type = model_type
+        self._model_type = model_type
         self.grain_column = grain_column
         self.prediction_column = prediction_column
-        self.y_pred = y_pred
-        self.y_actual = y_actual
+        self.test_set_predictions = test_set_predictions
+        self.test_set_actual = test_set_actual
         self._metric_by_name = metric_by_name
```

Comment:
I like that these live in the test set as well. Is it possible to get all the predictions back in the development step?